clients/google-api-services-admin/directory_v1/1.30.1/README.md

```diff
@@ -39,6 +39,6 @@ dependencies {
 }
 ```
 
-[javadoc]: https://googleapis.dev/java/google-api-services-admin-directory/latest/index.html
+[javadoc]: https://googleapis.dev/java/google-api-services-admin/latest/index.html
```

Comment:
google-api-services-admin has 3 different clients in it. datatransfer_v1, directory_v1, reports_v1. Looks like this link only points to reports.